pkg/configs/client/configs.go

```diff
@@ -33,8 +33,8 @@ func configsFromJSON(body io.Reader) (*ConfigsResponse, error) {
 }
 
 // GetLatestConfigID returns the last config ID from a set of configs.
-func (c ConfigsResponse) GetLatestConfigID() configs.ID {
-	latest := configs.ID(0)
+func (c ConfigsResponse) GetLatestConfigID(min configs.ID) configs.ID {
```

Comment:
should explain the function of `min` in the documentation (if you keep it, which I think you shouldn't)